pkg/network/ebpf/c/protocols/postgres/types.h

```diff
@@ -0,0 +1,31 @@
+#ifndef __POSTGRES_TYPES_H
+#define __POSTGRES_TYPES_H
+
+#include "conn_tuple.h"
+
+// Controls the number of Postgres transactions read from userspace at a time.
+#define POSTGRES_BATCH_SIZE 25
+
+// Maximum length of postgres query to send to userspace.
+#define POSTGRES_BUFFER_SIZE 64
+
+// Maximum number of Postgres messages we can parse for a single packet.
+#define POSTGRES_MAX_MESSAGES 100
```

Comment:
Do we have any insight if those values are set correctly by staging or production traffic?

Reply:
no, that's a chicken and egg problem
one of the followup tasks will be to measure the assumptions I took